packages/playground/test-built-npm-packages/commonjs-and-jest/tests/wp.spec.ts

```diff
@@ -23,7 +22,6 @@ SupportedPHPVersions.forEach((phpVersion: string) => {
 				expect(response.httpStatusCode).toBe(200);
 				expect(response.text).toContain('My WordPress Website');
 			} finally {
-				await php.exit();
```

Comment:
@adamziel @bgrgicak Is this line needed? There isn't an equivalent in the PHPWorker interface which is effectively what the `cli.playground` property exposes. My guess is that it is fine if we just close the server here since the playground.request() promise has already settled at this point.

Reply:
Apparently, that line is needed. One of the tests is running indefinitely.

https://github.com/WordPress/wordpress-playground/actions/runs/15790316388/job/44515285425?pr=2231#step:10:63

Reply:
This isn't quite right yet. The remote call of `playground[Symbol.asyncDispose]` wasn't succeeding, probably due to the method not being reflected in the comlink proxy.